.travis.yml

```diff
@@ -37,14 +37,17 @@ matrix:
       - PRODUCT_TESTS='-g hdfs_impersonation'
       - CLUSTER_TYPE='singlenode-hdfs-impersonation'
     - env:
-      - PRODUCT_TESTS='-g hdfs_impersonation,authorization'
+      - PRODUCT_TESTS='-g hdfs_impersonation,authorization,storage_formats,cli'
       - CLUSTER_TYPE='singlenode-kerberos-hdfs-impersonation'
     - env:
       - PRODUCT_TESTS='-g ldap_cli'
       - CLUSTER_TYPE='singlenode-ldap'
     - env:
       - MVN_TESTS='-Dtest=TestLocalBinarySpilledQueries,TestPostgreSqlDistributedQueries,TestMySqlDistributedQueries'
       - MVN_MODULES='-pl presto-tests,presto-mysql,presto-postgresql'
+    - env:
+      - PRODUCT_TESTS='-g hdfs_no_impersonation,storage_formats,cli'
```

Comment:
Guys, I think we should reconsider if we really want to re-run all those tests 3 times (singlenode, singlenode-hdfs-impersonation, singlenode-kerberos-hdfs-no-impersonation).

Also, I've run presto checks [with this branch](https://travis-ci.org/Teradata/presto-checks/jobs/183611610) and the tests themselves took 2:35. It'd have been 3 secs (for 3 or so tests) if not for the re-tested `storage_formats,cli` groups. The whole build took 8:14. Of all this, almost 5 minutes is just container spinup; the artifact download is 50 s. I'd like to raise the question: are those tests **that** critical, given our limited Travis resources? As I understand this tests that... cli and hive access work with kerberos on, with impersonation off?

(The unlimited workers power-up is going to wear up someday...)

I'd really like to see a better way of testing kerberos and auth mechanisms in general.

@mattsfuller what's your opinion?